pkgs/development/libraries/hpp-fcl/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, boost
+, eigen
+, assimp
+, octomap
+, qhull
+, pythonSupport ? false
+, python3Packages
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "hpp-fcl";
+  version = "2.3.3";
+
+  src = fetchFromGitHub {
+    owner = "humanoid-path-planner";
+    repo = finalAttrs.pname;
+    rev = "7e8fde64a5d2c2412325f6cb5d78623bf2409176";
```

Comment:
```suggestion
    rev = "v${finalAttrs.version}";
```

Reply:
The hash might also need to be updated.

Reply:
`v2.3.3` has failing unittests with nix, ref commit message and upstream https://github.com/humanoid-path-planner/hpp-fcl/pull/424

> on https://github.com/humanoid-path-planner/hpp-fcl/commit/7e8fde64a5d2c2412325f6cb5d78623bf2409176 instead of https://github.com/humanoid-path-planner/hpp-fcl/releases/tag/v2.3.3
> to fix a unittest where nix env set `boost::archive::tmpdir()` to `/build` and trigger a path concatenation bug.

Reply:
Is it possible to add this patch using `fetchpatch`? Otherwise, the version should be `unstable-2023-05-20`, which is less nice.

Reply:
Done !